recipe/build.sh

```diff
@@ -3,53 +3,98 @@
 # unset the SUBDIR variable since it changes the behavior of make here
 unset SUBDIR
 
-USE_NONFREE=no
-
 declare -a _CONFIG_OPTS=()
 
+# We choose the following flags because we don't enable NON-FREE features
 # We do not care what the defaults are as they could change. Be explicit
 # about every flag.
-if [[ ${USE_NONFREE} == yes ]]; then
-  _CONFIG_OPTS+=("--enable-nonfree")
-  _CONFIG_OPTS+=("--disable-gpl")
-  _CONFIG_OPTS+=("--disable-gnutls")
-  _CONFIG_OPTS+=("--enable-openssl")
-  # The Cisco GPL-compliant wrapper (you need to get your own binaries for this)
-  _CONFIG_OPTS+=("--enable-libopenh264")
-  # GPL-3.0
-  _CONFIG_OPTS+=("--enable-libx264")
-else
-  _CONFIG_OPTS+=("--disable-nonfree")
-  _CONFIG_OPTS+=("--enable-gpl")
-  _CONFIG_OPTS+=("--enable-gnutls")
-  # OpenSSL 3 will be Apache-licensed so we can revisit this later:
-  # https://github.com/openssl/openssl/commit/151333164ece49fdba3fe5c4bbdc3333cd9ae66d
-  _CONFIG_OPTS+=("--disable-openssl")
-  # The Cisco GPL-compliant wrapper (you need to get your own binaries for this)
-  _CONFIG_OPTS+=("--enable-libopenh264")
-  # GPL-3.0
+_CONFIG_OPTS+=("--disable-nonfree")
+_CONFIG_OPTS+=("--enable-gpl")
+_CONFIG_OPTS+=("--disable-gnutls")
+_CONFIG_OPTS+=("--disable-gcrypt")
+# As of OpenSSL 3, the license is Apache-2.0 so we can enable this
+_CONFIG_OPTS+=("--enable-openssl")
+# The Cisco GPL-compliant wrapper (you need to get your own binaries for this)
+if [[ ${target_platform} != win-64 ]]
+then
+_CONFIG_OPTS+=("--enable-libopenh264")
+_CONFIG_OPTS+=("--enable-libopus")
+_CONFIG_OPTS+=("--enable-libopenjpeg")
+_CONFIG_OPTS+=("--enable-libvorbis")
+_CONFIG_OPTS+=("--enable-pthreads")
+fi
+
+# enable other codecs and formats depending on platform
+# temporarily disabling librsvg because pkg-config doesn't find xau which is supposedly in the dependency chain of librsvg
+if [[ ${target_platform} != linux-64 ]] && [[ ${target_platform} != linux-aarch64 ]] && [[ ${target_platform} != linux-s390x ]] && [[ ${target_platform} != win-64 ]]
+then
+  _CONFIG_OPTS+=("--enable-librsvg")
+fi
+_CONFIG_OPTS+=("--enable-libtheora")
+
+_CONFIG_OPTS+=("--enable-libxml2")
+if [[ ${target_platform} != linux-s390x ]] && [[ ${target_platform} != win-64 ]]
```

Comment:
Please arrange the _CONFIG_OPTS in a way that makes it easy to see what's enabled on which platform and what's not.

Reply:
I'll do my best. Given this is a collage of what's available on main, what conda-forge is doing and what our licensing policies allow, it's not the most trivial thing to keep everything neat.

Reply:
done.